evap/staff/templates/staff_evaluation_operation.html

```diff
@@ -53,11 +53,59 @@
                 {% endif %}
             </div>
         {% endif %}
-        <input type="hidden" name="target_state" value="{{ target_state }}" />
+        <input type="hidden" name="target_state" value="{{ target_state }}"/>
+
+        {% if show_delete_answers_checkbox %}
+            <div class="card mb-3">
+                <div class="card-body">
+                    <div class="form-check">
+                        <input class="form-check-input" type="checkbox" name="delete-previous-answers" id="delete-previous-answers" checked>
+                        <label class="form-check-label" for="delete-previous-answers">
+                            {% trans "Delete previous answers" %}
+                        </label>
```

Comment:
If possible, I favor nesting inputs and labels instead of giving out `id`s. By putting the `input` inside the `label`, they also become associated without having to use `id` and `for`. (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)